app/src/main/java/bg/zahov/app/data/model/MeasurementType.kt

```diff
@@ -1,25 +1,71 @@
 package bg.zahov.app.data.model
 
+/**
+ * Enum class representing different types of body measurements.
+ * Each measurement type has an associated key that describes the measurement in a user-friendly format.
+ *
+ * @property key A descriptive name for the measurement type.
+ */
 enum class MeasurementType(val key: String) {
+    /** Represents the user's weight measurement. */
     Weight("Weight"),
+
```

Comment:
Again for strings use stringResources

Reply:
dont quite understand what i should do. Should i change the string in the data class to the id of the stringResources or ?

Reply:
change it